recipes/tensorflow-hub/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "0.1.0" %}
+
+package:
+  name: tensorflow-hub
+  version: {{ version }}
+
+build:
+  number: 0
+  # install using pip from the whl files on PyPI
+  # b/c this uses bazel to build.
+  script: python -m pip install --no-deps --ignore-installed https://pypi.io/packages/py2.py3/t/tensorflow_hub/tensorflow_hub-{{ version }}-py2.py3-none-any.whl
```

Comment:
This should be changed to
```
url: https://pypi.io/packages/py2.py3/t/tensorflow_hub/tensorflow_hub-{{ version }}-py2.py3-none-any.whl
script: python -m pip install --no-deps --ignore-installed tensorflow_hub-{{ version }}-py2.py3-none-any.whl
```
and license_file changed to.
```
license_file: tensorflow_hub-{{ version }}.dist-info/LICENSE.txt
```
(conda-build downloads the wheel into SRC_DIR and extracts it into SRC_DIR as well. Therefore the wheel can be installed and also files inside it like the license file can be used.)

Reply:
I guess you mean an actual `source` section, right? the as far as I know the `build` section cannot take the URL.

I did not do that b/c I though that `conda-build` would try to extract the wheel. Good to know that it does not.